# index the robot was registered with
int8 index

# active
bool active

# status message
string status

# active time (percentage)
float32 active_time

# number of used maintenances
int8 maintenance_count

# robot position and orientation
float32 x
float32 y
float32 yaw
